docs/io/unified_table.rst

```diff
@@ -28,10 +28,26 @@ then call the :class:`~astropy.table.Table`
 :meth:`~astropy.table.Table.read` method with the name of the file and
 the file format, for instance ``'ascii.daophot'``:
 
-.. doctest-skip::
-
+.. testsetup::
+    >>> import os
+    >>> with open('photometry.dat', 'w') as f: # doctest: +IGNORE_OUTPUT
+    ...     f.write("#N ID    XCENTER   YCENTER\n")
+    ...     f.write("#U ##    pixel     pixel \n")
+    ...     f.write("#F %-9d  %-10.3f   %-10.3f\n")
+    ...     f.write("#\n")
+    ...     f.write("14       138.538   256.405\n")
+    ...     f.write("18       18.114    280.170\n")
     >>> from astropy.table import Table
     >>> t = Table.read('photometry.dat', format='ascii.daophot')
+    >>> t.write('table.tex', format='latex')
+
+>>> from astropy.table import Table
```

Comment:
Indentation? Though now I see that this part *is* shown in the docs so perhaps one needs to make sure the indentation is different...